monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -530,7 +526,10 @@ def generate_report(cls):
                 "scanned": scanned_nodes,
                 "exploited_cnt": exploited_cnt,
             },
-            "recommendations": {"issues": issues},
+            "recommendations": {
+                "issues": issues,
+                "remediation_suggestions": remediation_suggestions,
```

Comment:
shouldn't "remediation_suggestion" be a part of issue? It looks like we need issue set to generate remediation suggestions that then will have to be accessed from a separate collection. If we are doing a data transfomation here, might as well just put the remediation into the issue.

Reply:
What is the difference?